src/utils/defaultTexts.ts

```diff
@@ -22,6 +22,16 @@ const defaultTexts = {
 
   //OptionList.tsx
   optionListTitle: 'Company Options',
+
+  //ProductShowcaseGrid
+  productSGTitle: 'Product',
```

Comment:
SG means what? More descriptive names make the hell of a difference down the road. Especially when you write this key once and then intelliSense takes over